internal/dao/stubs/account_dao.go

```diff
@@ -0,0 +1,79 @@
+package stubs
+
+import (
+	"context"
+	"errors"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/pointer"
+)
+
+var notInStoreError = errors.New("The Account you've been looking for does not exist")
+
+type accountDaoStub struct {
+	store []*models.Account
+}
+
+func buildAccountDaoWithOneAccount() *accountDaoStub {
+	return &accountDaoStub{
+		store: []*models.Account{
+			&models.Account{
+				ID:            1,
+				OrgID:         "1",
+				AccountNumber: pointer.String("1"),
+			},
+		},
+	}
+}
```

Comment:
Done ^_^

Reply:
So I'm back with dynamic, just in case, unless you think it's bad :)

Reply:
Just to make sure I understood: 
Do you that ipreferableeable to keep it dynamically because we mperformeform changes on static accounts which can lead failures in other tests? so we want our data to be revoked each time as a "clean slate"?

Reply:
Very good point, this must be dynamically created. Performance is not the issue here but concurrency is - we can easily run tests in parallel with Go and having this as a global variable would mean we share it - that would be bad.

Reply:
Now that I am thinking about it, it is worth enabling concurrent unit tests from the day one so if we run into concurrency issues we would solve them now rather than later with many tests.

Reply:
yes, exactly! :) Having always the same data at the beginning of the test is quite important for predictable test outcomes :)